landing-pages/site/content/en/community/_index.html

```diff
@@ -29,7 +29,7 @@
     </ol>
     {{< /accordion >}}
 
-    {{< accordion title="Report a bug" description="Use our [Jira](https://issues.apache.org/jira/projects/AIRFLOW/issues/) to create a ticket." logo_path="icons/bug-icon.svg" >}}
+    {{< accordion title="Report a bug" description="Use our [Github](https://github.com/apache/airflow/issues) to create a ticket." logo_path="icons/bug-icon.svg" >}}
```

Comment:
Should we change to github issue icon here?

Reply:
There was no JIRA icon here before, just a bug icon, so why there should be a Github icon?

Reply:
<img width="1265" alt="Screen Shot 2020-04-29 at 16 03 48" src="https://user-images.githubusercontent.com/15820530/80573741-219b1c00-8a33-11ea-9ef1-e8f43c7b831e.png">
I think we use "JIRA icon" named as default bug icon here

Reply:
Do you mean the footer?

<img width="246" alt="image" src="https://user-images.githubusercontent.com/5026382/80575758-bb9b9e00-8a0c-11ea-8938-741bb8f81bba.png">

Reply:
The bug icon is not a JIRA icon. I don't know where you saw it


<img width="250" alt="image" src="https://user-images.githubusercontent.com/5026382/80576879-9d36a200-8a0e-11ea-8c98-a3e8a932908d.png">

Reply:
Oh, yeah, I make a mistake, could you change the footer too?

Reply:
I removed the JIRA icon from the footer, since Github icon is already in the footer